atom/browser/net/url_request_async_asar_job.cc

```diff
@@ -38,6 +38,20 @@ void BeforeStartInUI(base::WeakPtr<URLRequestAsyncAsarJob> job,
     error = net::ERR_NOT_IMPLEMENTED;
   }
 
+  // sanitize custom headers
+  if (request_options && request_options->is_dict()) {
+    base::Value* headersDict = request_options->FindDictKey("headers");
```

Comment:
Can this be `const`?

```suggestion
    const base::Value* headersDict = request_options->FindDictKey("headers");
```

Reply:
Yes it can - but does that make a huuuge difference? ^^